### PR TITLE
[14.0] [MIG] mrp_multi_level

### PR DESCRIPTION
**Pull request by DavidBForgeFlow**
Standard migration to v.14.

Depends on OCA/manufacture #590

### CODE REVIEW

**Comment by HviorForgeFlow** [comment]
Superseeds https://github.com/OCA/manufacture/pull/583

File: mrp_multi_level/wizards/mrp_multi_level_views.xml [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
-    />
+<!--    <act_window-->
+<!--        name="Run MRP"-->
+<!--        res_model="mrp.multi.level"-->
+<!--        binding_model="mrp.multi.level"-->
+<!--        view_mode="form"-->
+<!--        target="new"-->
+<!--        id="action_mrp_multi_level"-->
+<!--    />-->
 </odoo>
```
Comment (LoisRForgeFlow) [inline mrp_multi_level/wizards/mrp_multi_level_views.xml]: convert this to `<record ... model="ir.actions.act_window" ...`

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Commit history is lost. Follow process in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Review by LoisRForgeFlow — APPROVED** [review]
All looks good now!

**Review by MateuGForgeFlow — APPROVED** [review]

**Pull request merged**